esbuild.config.mjs

```diff
@@ -131,6 +131,7 @@ THE SOFTWARE.
 `;
 
 const prod = (process.argv[2] === 'production');
+const dev = process.argv[2] === 'development';
```

Comment:
Whilst reading this, it occurred to me that there are some nice features in here, and to share the knowledge, it would be nice to have a line or two in CONTRIBUTING.md telling how to use what is available.

Reply:
Docs added in CONTRIBUTING - thank you.